packages/forms/src/components/ui/checkbox/Checkbox.tsx

```diff
@@ -11,36 +10,46 @@ import { Icon } from "@stenajs-webui/elements";
 import * as React from "react";
 import { ChangeEvent, useCallback, useEffect, useRef } from "react";
 import { FullOnChangeProps } from "../types";
+import { useId } from "../utils/UseId";
 import { CheckboxTheme, defaultCheckboxTheme } from "./CheckboxTheme";
 
 export interface CheckboxProps
   extends FullOnChangeProps<boolean, ChangeEvent<HTMLInputElement>>,
-    InputProps<HTMLButtonElement> {
+    InputProps<HTMLLabelElement> {
   disabled?: boolean;
   indeterminate?: boolean;
+  id?: string;
   theme?: CheckboxTheme;
 }
 
 const InvisibleCheckbox = styled.input`
   top: 0;
   left: 0;
-  width: 26px;
   cursor: inherit;
-  height: 26px;
   margin: 0;
   opacity: 0;
   padding: 0;
   position: absolute;
 `;
 
+const CheckboxLabel = styled.label`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  ${InvisibleCheckbox}:focus + & {
```

Comment:
Custom focus highlight? Is this correct? We don't have it anywhere else.
Which we probably should, but it should probably be centralized in some way, to ensure that all components use the same highlight.